onebusaway-android/src/main/java/com/joulespersecond/seattlebusbot/PreferencesActivity.java

```diff
@@ -149,7 +161,12 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
             }
         } else if (preference.equals(analyticsPref) && newValue instanceof Boolean) {
             Boolean isAnalyticsActive = (Boolean) newValue;
-            GoogleAnalytics.getInstance(this).setAppOptOut(!isAnalyticsActive);
+            //Report if the analytics turns off, just before shared preference changed
+            if (!isAnalyticsActive)
+                ObaAnalytics.reportEventWithCategory(ObaAnalytics.ObaEventCategory.APP_SETTINGS.toString(),
+                        getString(R.string.analytics_action_edit_general),
+                        getString(R.string.analytics_label_analytic_preference)
+                                + (isAnalyticsActive ? "YES" : "NO"));
```

Comment:
@barbeau I thought the same, since ```dispatchLocalHits()``` doesn't work with google play services. But as you mention, It's nice to have for consistency. I will update shortly.